### PR TITLE
fixing jshint single quotes issues

### PR DESCRIPTION
**Pull request by jloveland**
fixing jshint issues by requiring single quotes

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -12,6 +12,7 @@
   "undef": true, // Require all non-global variables be declared before they are used.
   "unused": false, // Warn unused variables.
   "strict": true, // Require `use strict` pragma in every file.
+  "quotmark": "single", // Require only single quotes
```
Comment (jloveland) [inline .jshintrc]: @lirantal and @ilanbiala , can we keep quotmark in here until #763 is addressed? Reason being is that I missed a lot of the double quote issues in my last PR.

Comment (ilanbiala) [inline .jshintrc]: @jloveland I don't think it's really that necessary to just make a temporary commit..I'd rather wait a few days for ESLint.

**Comment by lirantal** [comment]
Thanks @jloveland 
Were you able to actually reproduce the error where this happens? I remember reading in the other PR thread that it didn't happen to you...

**Comment by jloveland** [comment]
@lirantal, without `"quotmark": "single",` in the `.jshintrc` file, it doesn't check for single quotes. Will ESLint fox this?

**Comment by lirantal** [comment]
Yes, whenever @rhutchison will move forward with that ESLint PR...

**Comment by lirantal** [comment]
Since we are embracing the single quotes all around the project then this PR LGTM, and once we have linting for code styles then this will be enforced too.

@ilanbiala do you have any other thoughts about this?

File: modules/users/client/directives/password-validator.client.directive.js [hunk 2]
```diff
@@ -11,11 +11,11 @@ angular.module('users')
 
           // Strength Meter - visual indicator for users
           var strengthMeter = [
-            { color: "danger", progress: "20" },
-            { color: "warning", progress: "40"},
-            { color: "info", progress: "60"},
-            { color: "primary", progress: "80"},
-            { color: "success", progress: "100"}
+            { color: 'danger', progress: '20' },
```
Comment (ilanbiala) [inline modules/users/client/directives/password-validator.client.directive.js]: consistent spacing in this array of objects please

**Comment by ilanbiala** [comment]
Nope, LGTM.

**Comment by jloveland** [comment]
@ilanbiala I fixed spacing
@lirantal, without `"quotmark": "single`", in the `.jshintrc` file, it doesn't check for single quotes when I run `grunt jshint`. I can add it back in or we can wait for @rhutchison ESlint.
Let me know what you prefer and I can  squash if it's ready to merge.

**Comment by ilanbiala** [comment]
@jloveland we are going to add ESLint, so no need to add it back.

**Comment by lirantal** [comment]
@ilanbiala looks like we're delaying quite a bit with the ESLint PR so I recommend if we don't get that PR in the next few days from @rhutchison then let's just create it ourself. It is much more important now that every PR needs to conform to some kind of style.

**Comment by codydaig** [comment]
@lirantal I know @rhutchison has started on it. But hasn't been touched in a little bit. 

https://github.com/gym/mean/tree/eslint

**Comment by lirantal** [comment]
I remember we already have some issue which discusses code formatting but if we don't lets open one and put everything there for a discussion.

I'll merge this one in.

**Pull request merged**